### PR TITLE
Pre release backend

### PR DESCRIPTION
**Pull request by anpolimus**

### CODE REVIEW

**Comment by anpolimus** [comment]
@ivan-berezhnov https://geekgoddess.com/how-to-resize-the-google-nocaptcha-recaptcha/
We need this at our form :)

**Comment by anpolimus** [comment]
@ivan-berezhnov we need to change edit link little bit
image

**Comment by ivan-berezhnov** [comment]
@anpolimus please check my changes
screen shot 2019-02-15 at 2 33 22 am

**Comment by anpolimus** [comment]
@ivan-berezhnov we need prettify recaptcha also at this pr

File: config/sync/captcha.settings.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-default_challenge: captcha/Math
-description: 'This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.'
+default_challenge: recaptcha/reCAPTCHA
+description: Pl
```
Comment (podarok) [inline config/sync/captcha.settings.yml]: looks wrong

**Comment by anpolimus** [comment]
@podarok this is obfuscated sql for start. We dont store content and users data at this sql.
Just for quick local setup.

**Comment by anpolimus** [comment]
@podarok thank you that looking here.

**Pull request merged**